src/Pakettikauppa/Client.php

```diff
@@ -204,12 +209,8 @@ private function createShipment(Shipment &$shipment, $draft = false, $language =
             $shipment_xml->{"ROUTING"}->{"Routing.Comment"} = $this->comment;
         }
 
-        if (!$draft) {
-            $response = $this->doPost("/prinetti/create-shipment?lang={$language}", null, $shipment_xml->asXML());
-        } else {
-            $response = $this->doPost('/prinetti/create-shipment-draft', null, $shipment_xml->asXML());
-        }
-
+        $response = $this->doPost("/prinetti/create-shipment?lang={$language}", null, $shipment_xml->asXML());
```

Comment:
Draft shipment feature is deprecated, but if a draft shipment is attempted to be created, it should throw an error instead of creating a normal shipment.

Reply:
It thorws an exception now